switchboard/dds_util.py

```diff
@@ -303,9 +310,15 @@ def get_context(self):
         to_user = self.get_to_user()
         project = self.get_project()
         project_url = self.get_project_url()
+        try:
+            transfer_id = self.delivery.transfer_id or None
```

Comment:
I like how this try/except AttributeError is really simple. The one in `get_project` above is kind of concerning that we might sweep bugs under the rug. Could this be refactored into a method used by both `get_project` and `get_context`?